app/_includes/md/plugins-hub/queue-parameters.md

```diff
@@ -0,0 +1,62 @@
+<!-- shared with plugins that use queues. -->
+
+This plugin utilizes a queue to decouple the production and
+consumption of data.  This reduces the number of concurrent requests
+made to the upstream server under high load situations and provides
+for buffering during temporary network or upstream outages.
+
+A number of parameters can be tuned to influence the behavior and
+capacity of the queues used by the plugin.  Please see [XXX](the
+documentation on queues) for detail information on the usage of these
+parameters.
+
+The queue parameters all reside in a record under the key `queue` in
+the `config` parameter section of the plugin.
+
+parameters:
```

Comment:
Did you want this line and below to display like the table like the rest of the parameters? I think it would have to live in the same spot as those other parameters (I don't think it will render in this section).

Reply:
Can we somehow make the queue parameters into a separate table for each plugin?

Reply:
@hanshuebner I asked about making the parameters into a separate table and it's not possible. The queue parameters have to be included in each plugin’s index file manually as part of the params section.